src/style.css

```diff
@@ -1 +1,65 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  padding-top: 11px;
+  padding-bottom: 11px;
+  margin-right: 50px;
```

Comment:
just make a 50px padding both on the left and right sides of your container so you wouldn't need to add this margins
```suggestion
  padding: 0 50px;
```

Reply:
Я не можу цього зробити, так як тоді тести з Nav_tag не проходять, там позначено, що паддінги лише зверху і знизу, тому збоку я додала марджин

Reply:
<img width="1349" alt="Screenshot 2022-05-11 at 17 40 41" src="https://user-images.githubusercontent.com/75496546/167877211-0bfb2ed9-e618-4689-abbe-594d2fb2eaa1.png">

Reply:
Try to implement it this way

Reply:
Зробила і тоді тести з Nav так само не проходять